submarine-workbench/workbench-web/src/app/services/user.service.ts

```diff
@@ -96,6 +96,7 @@ export class UserService {
 
   createUser(sysUser: Partial<SysUser>): Observable<SysUser> {
     const apiUrl = this.baseApi.getRestApi('/sys/user/add');
+    sysUser['password'] = md5(sysUser.password);
```

Comment:
@charlie0220 When creating a user, you need to encrypt the user's password.
When the user logs in, the user’s password needs to be encrypted, and then corrected with the encrypted password in the database.